arm/Microsoft.SignalRService/webPubSub/deploy.bicep

```diff
@@ -0,0 +1,161 @@
+@description('Optional. The location to deploy the Web PubSub service.')
+param location string = resourceGroup().location
+
+@description('Required. The name of the Web PubSub resource.')
+param name string
+
+@description('Optional. Configuration Details for private endpoints. For security reasons, it is recommended to use private endpoints whenever possible.')
+param privateEndpoints array = []
+
+@allowed([
+  ''
+  'CanNotDelete'
+  'ReadOnly'
+])
+@description('Optional. Specify the type of lock.')
+param lock string = ''
+
+@description('Optional. Array of role assignment objects that contain the \'roleDefinitionIdOrName\' and \'principalId\' to define RBAC role assignments on this resource. In the roleDefinitionIdOrName attribute, you can provide either the display name of the role definition, or its fully qualified ID in the following format: \'/providers/Microsoft.Authorization/roleDefinitions/c2f4ef07-c644-48eb-af81-4b1b4947fb11\'.')
+param roleAssignments array = []
+
+@description('Optional. Tags of the resource.')
+param tags object = {}
+
+@description('The unit count of the resource. 1 by default.')
+param capacity int = 1
+
+@allowed([
+  'Free_F1'
+  'Standard_S1'
+])
+@description('Optional. Pricing tier of App Configuration.')
+param sku string = 'Free_F1'
```

Comment:
```suggestion
param sku string = 'Standard_S1'
```

Reply:
Note: https://github1s.com/Azure/azure-policy/blob/master/built-in-policies/policyDefinitions/SignalR/SignalR_AllowedSKU_AuditDeny.json

Reply:
It seems the anything but the `Free_F1` must be used to enable private endpoints

Reply:
Right, but do we need to make Standard_S1 the default? I like going with least surprises for the customer in regards to pricing :)

Reply:
We do if we want to enable a consumer to deploy the module in a secure way which should be the intention. 

In this sense, the cheapest option is not necessarily the best.

Reply:
Ok, everything else is done so far, can you please check again if anything is still missing?